pkgs/applications/misc/msglint/default.nix

```
@@ -0,0 +1,27 @@
+{ stdenv, fetchurl }:
+
+stdenv.mkDerivation {
+  pname = "msglint-1.04";
+  version = "1.04";
+
+  sourceRoot = ".";
+
+  src = fetchurl {
+    url = "https://tools.ietf.org/tools/msglint/msglint-1.04.tar.gz";
```

You could use the `version` number from above within the URL. Therefore you must at first add `rec` between `mkDerivation` and the brace in line 3.

```suggestion
    url = "https://tools.ietf.org/tools/msglint/msglint-${version}.tar.gz";
```